src/app/features/notification/notification-rule/notification-condition/notification-condition-editor/notification-condition-editor.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-dialog-content class="mat-typography">
+  @if (data.closeButton) {
+    <app-dialog-close mat-dialog-close></app-dialog-close>
+  }
+  <div class="json-editor">
+    <app-json-editor
+      [value]="jsonData"
+      (valueChange)="onJsonChange($event)"
+      (isValidChange)="onJsonIsValidChange($event)"
+    ></app-json-editor>
+
+    <div class="flex-row json-editor-actions">
+      <button mat-raised-button (click)="onJsonValueCancel()" i18n>
+        Cancel
+      </button>
+      <button
+        [disabled]="!isValidJson"
+        mat-raised-button
+        (click)="onJsonValueSave()"
+        i18n
+      >
+        Save
+      </button>
+    </div>
```

Comment:
we generally have "Save" first, "Cancel" second in order

Reply:
I have updated it and made the changes accordingly.